web-client/app/styles/main.scss

```diff
@@ -1153,6 +1153,7 @@ ul.cards-list-container {
         margin: 10px;
         display: block;
         padding-top: 1em;
+        overflow: scroll;
```

Comment:
sgtm. I am on a mac and chrome. I am seeing the broken highlight.js but pre seems to break words instead of continuing on a single line.

Anyway, will merge this and will close the bug #190 once it is deployed.

Will open another one to investigate highlightjs breakage.